framework/source/class/qx/ui/core/MExecutable.js

```diff
@@ -82,7 +82,6 @@ qx.Mixin.define("qx.ui.core.MExecutable",
      */
     _bindableProperties :
     [
-      "enabled",
```

Comment:
BTW: `qx.ui.form.CheckBox` overrides the `_bindableProperties` member in order to avoid forwarding of the icon property from the command to the widget. See https://github.com/qooxdoo/qooxdoo/blob/master/framework/source/class/qx/ui/form/CheckBox.js#L98-L109